src/app/components/Auth/AuthContainer.js

```diff
@@ -0,0 +1,77 @@
+import React, { PropTypes } from 'react';
+import { withRouter } from 'react-router';
+import * as firebase from 'firebase';
+import { connect } from 'react-redux';
+import { setAuthState } from '../../redux/actions.js';
+import Auth from './Auth.js';
+
+class AuthContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.handleRegister = this.handleRegister.bind(this);
+        this.handleLogin = this.handleLogin.bind(this);
+    }
+
+    handleRegister(email, password) {
```

Comment:
Haha, I'll make an issue for auth enhancement, but it probably won't get done for a while